tests/e2e/e2e_common.go

```diff
@@ -96,6 +96,7 @@ const (
 	poll                                       = 2 * time.Second
 	pollTimeout                                = 5 * time.Minute
 	pollTimeoutShort                           = 1 * time.Minute
+	healthStatusPollTimeout                    = 15
```

Comment:
Is this 15 minutes ? can you add a comment or change it to 15 * time.Minute

Reply:
I have created 2 variables `healthStatusPollInterval` and `healthStatusPollTimeout`, for 15 minutes and 15 seconds